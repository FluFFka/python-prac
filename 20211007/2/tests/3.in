45+3.5j, 23432-1111j
