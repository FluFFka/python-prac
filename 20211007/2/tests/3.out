(-23387+1114.5j)
